src/helpers/emailVerification.js

```diff
@@ -0,0 +1,46 @@
+const nodemailer = require('nodemailer');
+const bunyan = require('bunyan');
+
+module.exports = (username, email, token) => {
+  let transporter = nodemailer.createTransport({
+    service: 'Gmail',
+    auth: {
+      user: 'echo.annafreud@gmail.com',
+      pass: 'FAC!echo1'
+    },
+    logger: bunyan.createLogger({
+      name: 'nodemailer'
+    }),
+    debug: true
+  }, {
+    from: 'Echo do-not-reply <echo.annafreud@gmail.com>',
+  });
+
+  console.log('SMTP Configured');
+
+  let message = {
+    to: `${username} <${email}>`,
+    subject: 'Please confirm your account',
+    text: 'Please confirm your account',
+    html: `<p>Hi ${username}, welcome to echo
+    <br> Please confirm your account by visiting the following link: <br><a href="http://localhost:3000/verify/${token}/${username}">http://localhost:3000/verify/${token}/${username}</a>
```

Comment:
`localhost:3000` put this in env file or something.

You can actually get this information from the return value of app.listen()

This function returns http.server that inherited from net.server with has a method called address

![image](https://user-images.githubusercontent.com/12845233/28571734-3280debe-713c-11e7-9656-9b8c457036bc.png)

But not sure how you get this without some kind of circular dependency issue so if this seems overly complex feel free to just put the hostname in the .env file